src/lang/modifyAst/addFillet.ts

```diff
@@ -231,6 +322,89 @@ export function addFillet(
   return new Error('Unsupported extrude type.')
 }
 
+function getPathToNodeOfFilletLiteral(
+  pathToExtrudeNode: PathToNode,
+  extrudeDeclarator: VariableDeclarator,
+  tag: string
+): PathToNode {
+  let pathToFilletObj: any
```

Comment:
```suggestion
  let pathToFilletObj: PathToNode
```
Or maybe needs to be
```suggestion
  let pathToFilletObj: PathNoNode = []
```